site/content/en/docs/getting-started/installation.md

```diff
@@ -13,6 +13,17 @@ description: >
 helm install llmaz oci://registry-1.docker.io/inftyai/llmaz --namespace llmaz-system --create-namespace --version 0.0.9
 ```
 
+To enable InftyAI scheduler, please apply the following value file to the command above:
```

Comment:
I think we can move this to a new catagory: Features, and it should be `heterogeneous cluster support`.